test/test_remotes.rb

```diff
@@ -192,6 +192,26 @@ def test_modifies_from_many_threads
     end
   end
 
+  def test_mtime
+    Dir.mktmpdir 'test' do |dir|
+      file = File.join(dir, 'remotes')
+      FileUtils.touch(file)
+      remotes = Zold::Remotes.new(file: file)
+      sleep 2
+      remotes.add('127.0.0.1')
+      assert(Time.now - remotes.mtime <= 1)
```

Comment:
@emilianodellacasa this a very dangerous assumption. What if the build server is slow and this operation will actually take longer than a second?